yaml/_dumper.ts

```diff
@@ -533,7 +533,7 @@ function writeScalar(
   level: number,
   isKey: boolean,
 ) {
-  state.dump = ((): string => {
+  const createDump = () => {
```

Comment:
We typically reserve the arrow function syntax for anonymous functions. Changing...

Reply:
Yes, but since this functions is now a method of DumperState, you would need to bind `this` with a regular function, right?